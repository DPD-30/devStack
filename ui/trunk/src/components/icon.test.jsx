import React from 'react'
import { renderToString } from 'react-dom/server'


import TestComponent from './icon.jsx'

jest.unmock('./icon.jsx')


test('Font Awesome Icon matches snapshot for generic', () => {
	const comp = renderToString(<TestComponent name="foo" />)

	expect(comp).toMatchSnapshot()
})
test('Font Awesome Icon matches snapshot for special case hour-glass-half - f252', () => {
	const comp = renderToString(<TestComponent name="hour-glass-half - f252" />)

	expect(comp).toMatchSnapshot()
})
test('Font Awesome Icon matches snapshot for special case exclamation point - f12a', () => {
	const comp = renderToString(<TestComponent name="exclamation point - f12a" />)

	expect(comp).toMatchSnapshot()
})
