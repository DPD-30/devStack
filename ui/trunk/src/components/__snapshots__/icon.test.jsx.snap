// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Font Awesome Icon matches snapshot for generic 1`] = `
<span aria-hidden="true"
      class="fa fa-foo"
      data-reactroot
>
</span>
`;

exports[`Font Awesome Icon matches snapshot for special case exclamation point - f12a 1`] = `
<span aria-hidden="true"
      class="fa fa-exclamation point"
      data-reactroot
>
</span>
`;

exports[`Font Awesome Icon matches snapshot for special case hour-glass-half - f252 1`] = `
<span aria-hidden="true"
      class="fa fa-hourglass-half"
      data-reactroot
>
</span>
`;
